res/Sectors/M1105/Lorspane.txt

```diff
@@ -28,377 +28,376 @@
 # Subsector O: Ogoress
```

Comment:
The Hive Federation in Lorspane should have 8-10 subsector capitals, they currently have none.